app/src/main/java/org/mozilla/fenix/StrictModeManager.kt

```diff
@@ -120,11 +123,11 @@ class StrictModeManager(
             // This can overflow and crash. However, it's unlikely we'll suppress StrictMode 9
             // quintillion times in a build config where StrictMode is enabled so we don't handle it
             // because it'd increase complexity.
-            suppressionCount += 1
+            val suppressionCount = suppressionCount.incrementAndGet()
 
             // We log so that devs are more likely to notice that we're suppressing StrictMode violations.
             // We add profiler markers so that the perf team can easily identify IO locations in profiles.
-            logger.warn("StrictMode violation suppressed: #$suppressionCount")
+            logger.warn("StrictMode violation suppressed: #${suppressionCount}")
```

Comment:
nit (while we're changing this): violation(s)?

Reply:
The way I intended this is, "A StrictMode violation was suppressed RIGHT NOW. It's the X instance of it." I feeling like changing it to "violations" is less clear: did the violation happen just now? Did multiple violations happen at once? I'd rather clarify "this is the # X instance of it" but I'm not sure how to do that